code/game/objects/effects/anomalies.dm

```diff
@@ -31,10 +31,13 @@
 
 
 /obj/effect/anomaly/proc/anomalyNeutralize()
-	new /obj/effect/effect/bad_smoke(loc)
+	var/turf/T = get_turf(src)
 
-	for(var/atom/movable/O in src)
-		O.loc = src.loc
+	new /obj/effect/effect/bad_smoke(T)
+
+	if(aSignal)
+		aSignal.forceMove(T)
+		aSignal = null
```

Comment:
I don't think that'd be a good idea, object deletion shouldn't leave a trace, and what of the case if the anomaly "Goes off" and fails to be defused?